crates/jxl-oxide/tests/test_fuzz_findings.rs

```diff
@@ -0,0 +1,36 @@
+use jxl_oxide::JxlImage;
+
+fn fuzz_decode(data: &[u8]) {
+    if let Ok(mut image) = JxlImage::from_reader(std::io::Cursor::new(data)) {
+        let header = image.image_header();
+        let max_size = u32::max(header.size.width, header.size.height);
+
+        // Skip huge images
+        if max_size > 65536 {
+            return;
+        }
```

Comment:
For example, we don't need this part since we're not fuzzing here.

Reply:
Oh, fixed now. Thanks!